mod_reforged/hooks/entity/tactical/enemies/bandit_leader.nut

```diff
@@ -57,12 +57,14 @@
 			if (this.m.Items.hasEmptySlot(::Const.ItemSlot.Offhand))
 			{
 				local weapon = ::MSU.Class.WeightedContainer([
+					[1, "scripts/items/weapons/greataxe"],
+					[1, "scripts/items/weapons/greatsword"],
 					[1, "scripts/items/weapons/longsword"],
 					[1, "scripts/items/weapons/rf_kriegsmesser"],
 					[1, "scripts/items/weapons/rf_swordstaff"],
 					[1, "scripts/items/weapons/two_handed_flail"],
-					[1, "scripts/items/weapons/two_handed_flanged_mace"],
-					[1, "scripts/items/weapons/greatsword"]
+					[1, "scripts/items/weapons/two_handed_hammer"],
+					[1, "scripts/items/weapons/two_handed_flanged_mace"]
 				]).roll();
```

Comment:
If we go the route of that suggestion of creating an item table for top tier non-named melee weapons, how do you think would that affect the thematic-ness of these enemies?